cubano-webdriver-manager/src/main/java/org/concordion/cubano/driver/web/provider/LocalBrowserProvider.java

```diff
@@ -82,7 +82,7 @@ protected void setupBrowserManager(WebDriverManager instance) {
                             throw new RuntimeException("Unable to update last checked date", e);
                         }
                     } else {
-                        instance.forceCache();
+//                        instance.forceCache();
```

Comment:
@andrew-sumner @jimmykemp Please check this change. The forceCache() method has been removed and I can't see a substitute method. Do you have a test that will check whether it is OK to remove this line?